src/projects.rs

```diff
@@ -22,6 +22,21 @@ pub struct Project {
     pub kubernetes: Option<KubernetesDetails>,
 }
 
+impl Project {
+    pub fn default(id: String, description: String, script: String) -> Self {
```

Comment:
This should be called `new`. Generally if you have an actual default method, it will be implementing the `Default` trait. If any config is needed, `new` is recommended